src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+  font-family: $main-font;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #060b35;
+}
+
+.card {
+  padding: 16px;
+  padding-top: 32px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: $main-white-color;
```

Comment:
Ensure that the variable `$main-white-color` is defined. If it's not defined, it will cause a compilation error.